### DISCUSSION TITLE
What is the proper way to change theme... that actually works!

### DISCUSSION THREAD

**Discussion by jonomynard** [vakata/jstree#2720] [Q&A]
Hi all,

I'm a true novice so apologies in advance for the rookie question. I've been trying to change theme on a basic jstree (e.g. from default to default-dark).  In terms of how this is done, there seems to be no consistency across various forums.  The best I can gather is the following (editing jstree/demo/basic/index.html, from the out-of-the-box jstree-master code):

1. Change style sheet to default-dark, e.g.
`<link rel="stylesheet" href="./../../dist/themes/default-dark/style.min.css" />`

2. Specify theme change in core : themes, and (maybe?) include the "themes" plugin, as follows:

```javascript
$('#data').jstree({
   'plugins': ['themes'],
   'core' : {
	'themes': {
        'name': 'default-dark',
        'dots' : false,
        'responsive' : true,
	'icons' : false,
      	'stripes' : false
      },
     'data' : [
           { "text" : "Root node", "children" : [
		{ "text" : "Child node 1" },
		{ "text" : "Child node 2" }
            ]}
         ]
   }
});
``` 

However, this doesn't work for me. Nothing changes and the dots and icons are still visible.  Some forums have the "themes" information outside of "core", some appear to have "themes" as a plugin. Elsewhere, the theme is changed after creating the jstree instance (this also doesn't seem to work for me).  

Any help would be greatly appreciated. Thanks!

**Comment by vakata** [comment]
Here is a demo: https://jsfiddle.net/4kzq176u/

I included both themes and used your config - seems to work fine.
